theme/templates/resource-landing-page/modals.html

```diff
@@ -1573,7 +1573,7 @@ <h4 class="modal-title" id="delete-resource-title">Delete Resource</h4>
     {% endif %}
 
     {% if user.is_authenticated and rights.statement %}
-        {% if cm.raccess.public or self_access_level != 'None' %}
+        {% if can_copy %}
             {% if allow_resource_copy %}
```

Comment:
I think allow_resource_copy should be sufficient, so can_copy check is not needed.

Reply:
I did not see allow_resource_copy :).  But it doesn't get the desired behavior.  It checks if the user is logged in, if the user owns the resource and then rights statement.  Thanks for pointing this method out, I'll see if it can be updated to get the desired behavior without affecting other areas.

Reply:
looks like allow_resource_copy is specifically for checking the license according to https://github.com/hydroshare/hydroshare/blob/develop/hs_core/hydroshare/resource.py#L621-L623

Reply:
you can see the check to can_view_resource right above it

Reply:
OK. Do we have a can_view_resource context variable that can be used in place of can_copy for permission check? I am thinking can_view_resource is more generally applicable for other actions as well while can_copy is too specific and it looks like can_view_resource is equal to can_copy in terms of permission access control.

Reply:
I just pushed a commit that should clear this up